kubetest/e2e.go

```diff
@@ -392,8 +394,15 @@ func DumpClusterLogs(location string) error {
 }
 
 func DumpFederationLogs(location string) error {
-	log.Printf("Dumping Federation logs to: %v", location)
-	return finishRunning(exec.Command("./federation/cluster/log-dump.sh", location))
+	logDumpPath := "./federation/cluster/log-dump.sh"
+	// federation/cluster/log-dump.sh only exists in the Kubernetes tree
+	// post-1.6. If it doesn't exist, do nothing and do not report an error.
+	if i, err := os.Stat(logDumpPath); err == nil && !i.IsDir() {
```

Comment:
Fair question. But on the other hand, if the file is a `SymLink()` there are other problems. The sym link could be broken. It is probably not worth including all possible permutations of checks.

Anyway, I don't feel too strongly about this. It is fine either way.